ci-operator/step-registry/gather-network-trace/OWNERS

```diff
@@ -0,0 +1,27 @@
+approvers:
+- alexanderconstantinescu
+- danwinship
+- dcbw
+- knobunc
+- jacobtanenbaum
+- pecameron
+- rcarrillocruz
+- squeed
+reviewers:
+- wking
+- stevekuznetsov
```

Comment:
Can we please make the OWNERS files here as targeted as possible? I would not know how to review this.